config/Dockerfiles/tests.d/libvirt/04_libvirt-network-dns

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -xe
 
 # Verify basic provisioning for all supplied providers
-# distros.exclude: fedora28 centos7 fedora28
+# distros.exclude: centos7
```

Comment:
Can this be run on just one distro?  Also, you may want to exclude all distros and only include the libvirt provider as per the change you made in 03_libvirt-network